src/service-api.ts

```diff
@@ -7,18 +7,55 @@
  * in Graasp as a fastify server plugin
  */
 
-import { FastifyPluginAsync } from "fastify";
+import { FastifyPluginAsync } from 'fastify';
+import { ChatService } from './db-service';
+import { ChatMessage } from './interfaces/chat-message';
+import common, { getChat, publishMessage } from './schemas';
+import { TaskManager } from './task-manager';
 
 /**
  * Type definition for plugin options
  */
-interface GraaspChatboxPluginOptions {
+interface GraaspChatPluginOptions {
```

Comment:
Hmm. If we keep calling it `Chat`, shouldn't the repo be called `...-chat` too?

Reply:
Good point, these 2 repos are paired: https://github.com/graasp/graasp-plugin-chatbox and https://github.com/graasp/graasp-chatbox
However I felt that modeling the chat objects would be more semantically precise with the `Chat` name rather than the `Chatbox` (which seems to refer more to the UI side of things). We could rename both repos if needed.